wagtail_wordpress_import/test/tests/test_functions.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+from wagtail_wordpress_import.functions import snakecase_key, dict_to_list
+
+
+class TestSnakeCaseKey(TestCase):
+    def test_snake_case_key_remove_colon(self):
+        self.assertEqual(snakecase_key("foo:bar"), "foo_bar")
+
+    def test_snake_case_key_remove_leading_underscore(self):
+        self.assertEqual(snakecase_key("_foo_bar"), "foo_bar")
+
+
+class TestDictToList(TestCase):
+    def test_dict_to_list_with_dict(self):
+        node = {"foo": {"bar": "baz"}}
+        self.assertEqual(dict_to_list(node, "foo"), [{"bar": "baz"}])
+
+    def test_dict_to_list_with_list(self):
+        node = {"foo": [{"foo": "bar"}, {"foo": "baz"}]}
+        self.assertEqual(dict_to_list(node, "foo"), [{"foo": "bar"}, {"foo": "baz"}])
```

Comment:
**Suggestion**: Add null/empty examples here, too, if it's relevant. If you expect the function just to raise an exception, maybe that's also worth testing, so that changes to the function in future are caught by this as a regression test, and it forces the developer to add tests for their improved future behaviour.